client/go/commands/cluster_create.go

```diff
@@ -87,11 +85,7 @@ func (a *CreateNewClusterCommand) Exec(args []string) error {
 
 	//check status code
 	if r.StatusCode != http.StatusCreated {
-		s, err := utils.GetStringFromResponse(r)
-		if err != nil {
-			return err
-		}
-		return errors.New(s)
+		utils.GetStringFromResponseCheck(r)
```

Comment:
I'm not sure why this code changed.  The code that was removed is the correct method.  Please check all calls that pull the error string from the body.  They need to return it to the caller.